src/core/shape/vertex.js

```diff
@@ -2253,6 +2253,77 @@ p5.prototype.normal = function(x, y, z) {
   return this;
 };
 
+/** Sets the shader's vertex attribute variables.
+ * 
+ * Shader programs run on the computer's graphics processing unit (GPU)
+ * They live in a part of the computer's memory that's completely separate from
```

Comment:
I wonder if this might be a bit more context than is needed at first. I'm planning to update the tutorials in the Learn section of the site when we do the 2.0 release to mention attributes and shader hooks, that might be a better place to explain the memory model? I wonder if all users need to know here is that we're adding extra data per vertex that you can read in a shader.

Reply:
Yeah you might be right, to be honest I copied it from [`setUniform()`](https://p5js.org/reference/p5.Shader/setUniform/) but they could both change.